docs/sources/developers/kinds/composable/lokidataquery/schema-reference.md

```diff
@@ -0,0 +1,26 @@
+---
+keywords:
+  - grafana
+  - schema
+title: LokiDataQuery kind
+---
+> Both documentation generation and kinds schemas are in active development and subject to change without prior notice.
+
+# LokiDataQuery kind
+
+## Maturity: experimental
+## Version: 0.0
+
+## Properties
+
+| Property       | Type    | Required | Description                             |
+|----------------|---------|----------|-----------------------------------------|
+| `editorMode`   | string  | No       | Possible values are: `code`, `builder`. |
+| `expr`         | string  | No       | the LogQL query                         |
+| `instant`      | boolean | No       | @deprecated, now use queryType          |
+| `legendFormat` | string  | No       | used to override the name of the series |
+| `maxLines`     | integer | No       | limit the number of log rows returned   |
+| `range`        | boolean | No       | @deprecated, now use queryType          |
+| `resolution`   | integer | No       |                                         |
```

Comment:
Is there a description for resolution?

Reply:
thanks for looking into this, i added documentation for it ("Used to scale the interval value.")